templates/controllers/grid/navigationMenus/customNMIType.tpl

```diff
@@ -0,0 +1,28 @@
+{**
+ * templates/controllers/grid/navigationMenus/customNMIType.tpl
+ *
+ * Copyright (c) 2014-2018 Simon Fraser University
+ * Copyright (c) 2000-2018 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * Custom Custom NMI Type edit form part
+ *}
+<div id="NMI_TYPE_CUSTOM" class="NMI_TYPE_CUSTOM_EDIT">
+	{fbvFormSection id="targetPath"}
```

Comment:
Is the id `targetPath` necessary? If so, let's give it a more unique prefix, like `nmiTargetPath`.